stable/prometheus-operator/README.md

```diff
@@ -193,6 +193,7 @@ The following tables list the configurable parameters of the prometheus-operator
 | `prometheusOperator.admissionWebhooks.patch.image.pullPolicy` | Image pull policy for the webhook integration jobs | `IfNotPresent` |
 | `prometheusOperator.admissionWebhooks.patch.image.repository` | Repository to use for the webhook integration jobs | `jettech/kube-webhook-certgen` |
 | `prometheusOperator.admissionWebhooks.patch.image.tag` | Tag to use for the webhook integration jobs | `v1.2.1` |
+| `prometheusOperator.admissionWebhooks.patch.image.sha` | Sha to use for the webhook integration jobs (optional) | `3e6adf0140f5749427a83a750cc436c532872fa7bf01d01b25bb163737dab245` |
```

Comment:
Could we drop the default SHA configurations? Makes updates less of a headache and I'm not sure this is something that I would want the chart to provide for me if I wanted to set this configuration.

Reply:
Sure, change pushed.